pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -63,9 +63,9 @@ func newController(objects ...runtime.Object) (*fake.Clientset, *networkPolicyCo
 	addressGroupStore := store.NewAddressGroupStore()
 	internalNetworkPolicyStore := store.NewNetworkPolicyStore()
 	npController := NewNetworkPolicyController(client, informerFactory.Core().V1().Pods(), informerFactory.Core().V1().Namespaces(), informerFactory.Networking().V1().NetworkPolicies(), addressGroupStore, appliedToGroupStore, internalNetworkPolicyStore)
-	npController.podListerSynced = alwaysReady
-	npController.namespaceListerSynced = alwaysReady
-	npController.networkPolicyListerSynced = alwaysReady
+	//npController.podListerSynced = alwaysReady
```

Comment:
just checking: do we need to remove this because otherwise the `List` operations may not return the entire set of Pods / Namespaces?

do you know why we used `alwaysReady` in the first place, I can't remember?